import sys

def err(msg):
    sys.exit(msg)
